dandischema/models.py

```diff
@@ -926,7 +1000,7 @@ def contributor_musthave_contact(cls, values):
     wasGeneratedBy: Optional[List[Project]] = Field(
         None,
         title="Associated projects",
-        description="Describe the project(s) that generated this Dandiset",
+        description="Describe the project(s) that generated this Dandiset.",
```

Comment:
```suggestion
        description="Project(s) that generated this Dandiset.",
```

since in general non-imperative (shorter) form is used throughout